src/widget/weffectparameterknobcomposed.cpp

```diff
@@ -25,13 +25,16 @@ void WEffectParameterKnobComposed::setup(const QDomNode& node, const SkinContext
 }
 
 void WEffectParameterKnobComposed::parameterUpdated() {
+    std::optional<double> neutralPoint = std::nullopt;
     if (m_pEffectParameterSlot->isLoaded()) {
-        setBaseTooltip(QString("%1\n%2").arg(
+        setBaseTooltip(QStringLiteral("%1\n%2").arg(
                 m_pEffectParameterSlot->name(),
                 m_pEffectParameterSlot->description()));
+        neutralPoint = m_pEffectParameterSlot->neutralPointOnScale();
```

Comment:
If we have here a std:nullopt we have a skin error. Likely the a button slot is connected to a knob. 
Should we issue a warning here?

Reply:
Ok, I have not tested that case. That is sufficient than.